src/node/desktop/test/config/config.test.ts

```diff
@@ -23,16 +23,12 @@ import { BuildType } from '../../src/config/config';
 describe('Config', () => {
 
   it('Should reflect active configuration', async () => {
-
-    // since 'src/config' is a symlink to the 'active' build type,
-    // we resolve that for testing the 'active' build type
-    const configPath = fs.realpathSync('src/config');
-    if (configPath.includes('development')) {
-      expect(BuildType.Current).to.equal(BuildType.Development);
+    const contents = fs.readFileSync('src/config/config.ts', 'utf8');
```

Comment:
Since the config is a .js file, why not just import it and use it as code instead of reading it and parsing it?